src/main/java/org/cs1331/gitsubmitter/Application.java

```diff
@@ -32,7 +32,7 @@ private static boolean checkCompile(String[] files) throws Exception {
         return p.exitValue() == 0;
     }
 
-    private static StudentSubmission authenticateAndCreate(String repo)
+    public static StudentSubmission authenticateAndCreate(String repo)
```

Comment:
I've since fixed this but the diff is apparently not reflecting my updated changes. I also added a boolean flag `appendUsername`